src/transformState.js

```diff
@@ -5,7 +5,27 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const order of actions) {
```

Comment:
why does `order` is an element of `actions` array?

Reply:
Good advice !  Name of  the element  of actions array was change to more obvious one.